site/docs/components/static-list/usage.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Using the component
+
+### When to use
+
+- To present a set of list items which are static. It can be used in conjunction with File drop zone to create a file uploader pattern to show the list of files added, it can also be used to create a pattern for cart details.
```

Comment:
This description goes straight to File Dropdown and File Uploader pattern, which we do not have.
Can we remove that and as per description, reference them as example use-cases but not so specific.

Ah ... we use cart here but in my comment, I said 'checkout basket'.. Need to align that
So, we either use 'shopping cart' or 'checkout basket'.
I don't have a preference, as long as the language aligns.

It should be bias to US locale, does 'checkout basket' work in the US, or is 'shopping cart' more familiar?